requirements/prod.txt

```diff
@@ -61,7 +60,6 @@ mock==1.0.1
 monolith.client==0.9
 moz-addon-packager==1.0.1
 mozilla-logger==0.2
-mozpay==2.0.0
 m2secret==0.1.1
```

Comment:
It probably no longer needs M2Crypto. That would save you guys a lot of headache if you could remove that one.